test/spec/modules/yieldoneBidAdapter_spec.js

```diff
@@ -190,7 +190,8 @@ describe('yieldoneBidAdapter', function() {
         'ttl': 3000,
         'referrer': '',
         'mediaType': 'video',
-        'vastXml': '<!-- vast -->'
+        'vastXml': '<!-- vast -->',
+        'renderer': ''
```

Comment:
I would think that there could be a better unit test here checking that the renderer comes out to be img.ak.impact-ad.jp/ic/pone/ivt/firstview/js/dac-video-prebid.min.js ?

Reply:
Hello @bretg 
Thank you for reviewing.

we added unit test about renderer.
please confirm it.

Reply:
Hello @bretg 
Could you confirm this PR?